content/en/docs/Writing policies/mutate.md

```diff
@@ -485,3 +485,51 @@ Labels:       backup-needed=no
               type=database
 <snip>
 ```
+
+## foreach
+
+The `foreach` declaration simplifies mutation of sub-elements in resource declarations, for example Containers in a Pod. 
+
+A `foreach` must contain a `list` attribute that defines the list of elements it processes and a `patchStrategicMerge` declaration. For example, iterating over the list of containers in a Pod is performed using this `list` declaration:
+
+```yaml
+list: request.object.spec.containers
+patchStrategicMerge:
+  spec:
+    containers:
+       ...
+```
+
+When a `foreach` is processed, the Kyverno engine will expand fetch the list of elements specified by `list`.
```

Comment:
... expand fetch... ?

Reply:
`fetch` seems to be the wrong term. Will update to `evaluate` as we are actually doing a JMESPath expression evaluation on the context.